Main/Source/lterras.cpp

```diff
@@ -1465,15 +1504,18 @@ truth coffin::Open(character* Opener)
   truth Success = olterraincontainer::Open(Opener);
   if(Success)
   {
+    ADD_MESSAGE("You feel a horrible cruse spreading.");
```

Comment:
cruse should be curse (but we could use cruse...? lol nah j/k)

Reply:
u been curst bro!!1!!!!1